test/WearableUIGallery/WearableUIGallery.Tizen.Wearable/WearableUIGallery.Tizen.Wearable.csproj

```diff
@@ -19,6 +20,7 @@
 
   <!-- If solution already has PCL project, will reference -->
   <ItemGroup>
+    <ProjectReference Include="..\..\..\src\Tizen.Wearable.CircularUI.Chart.Forms.Renderer\Tizen.Wearable.CircularUI.Chart.Forms.Renderer.csproj" />
```

Comment:
Where is Tizen.Wearable.CircularUI.Chart.Forms.Renderer.csproj file ?  
I can't find following file.
"Tizen.Wearable.CircularUI.Chart.Forms.Renderer\Tizen.Wearable.CircularUI.Chart.Forms.Renderer.csproj"